views/index.html

```diff
@@ -15,6 +15,8 @@ <h1>Página principal</h1>
 
     <br>
     <br>
+    <a href="/login">Login</a> <br/>
+    <a href="/cadastroUser">Novo usuário</a> <br/>
```

Comment:
Não esqueça de ajustar aqui:

```suggestion
    <a href="/cadastro-user">Novo usuário</a> <br/>
```